catalog-info.yaml

```diff
@@ -0,0 +1,30 @@
+# Backstage documentation
+# https://backstage.io/docs/features/software-catalog/descriptor-format/
+
+# !!! WARNING !!!
+# This is a template file with a number of fields that need to be filled before merging this to the default branch
+apiVersion: backstage.io/v1alpha1
+# Component, API, Template, Group, User, Resource, System, Domain, Location
+kind: unknown
+metadata:
+  name: forma-36 
+  description: unknown
+  annotations:
+    github.com/project-slug: contentful/forma-36 
+    contentful.com/service-tier: "unknown" #1, 2, 3, 4
+
+  tags:
+    - update-me
+    #need to add sast.yaml to .github/workflows and enable it in polaris dashboard
+    #once that is done this can be changed to sast-enabled
+    - sast-disabled
+    #make this match the value from service-tier above
+    - tier-unknown
+spec:
+  #cli, component, contentful.com/template, documentation, function, library, service, template, website
+  type: unknown 
+  #deprecated, experimental, production, unknown
+  lifecycle: unknown  
+  system: unknown #optional
+  # your team name as it appears in github when tagging them for reviews
+  owner: group:team-unknown 
```

Comment:
@roryscarson Historically, the owner of Forma 36 is [bauhaus](https://github.com/orgs/contentful/teams/team-bauhaus) but this team "became" [core experience](https://github.com/orgs/contentful/teams/team-core-experience). Now a dedicated team was created again to own Forma 36, but the team is not available on GitHub :face_with_spiral_eyes:

Reply:
So this is handled via automation that pulls from workday. For this to be setup the team needs to get setup there first.  @contentful/team-mechagodzilla handles the association with github and can help set that up if the new team is already setup in workday, if they aren't setup there then for all intents and purposes they aren't considered a real team yet and that setup needs to be done or ownership of the repo to temporarily remain with an existing team.

Reply:
Let's put @contentful/team-core-experience for now, I will see if we can create a Forma 36 team on GitHub.